tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
 #   py26,py27,py33,py34,py35,pypy,pypy3,
-    py27,py34,py35,py36,py37,pypy
+    py27,py34,py35,py36,py37,py38,pypy
```

Comment:
I missed a couple things on the first pass.

[Python 3.4 is now EOLed](https://www.python.org/dev/peps/pep-0429/#release-schedule). Suggest its removal as well, both here and below.

Also the commented line 3 should be dropped.